docs/mobile-apps/features/biometric-authentication.md

```diff
@@ -49,6 +51,25 @@ Biometrics test support in Sauce Labs is not intended to test actual biometrics
 * Ephemeral apps.
 :::
 
+### Android Biometrics
+
+For Android devices, there are multiple ways to configure biometrics. From API 28 and above we support the following:
+
+* [`BiometricManager`](https://developer.android.com/reference/android/hardware/biometrics/BiometricManager) provides APIs to query if the app can authenticate and list of [Authenticators](https://developer.android.com/reference/android/hardware/biometrics/BiometricManager.Authenticators).
+* [`BiometricPrompt`](https://developer.android.com/reference/android/hardware/biometrics/BiometricPrompt) provides a user interface in the form of a dialog which is used the user to provide the finger touch and the call [`BiometricPrompt.AuthenticationCallback`](https://developer.android.com/reference/android/hardware/biometrics/BiometricPrompt.AuthenticationCallback) provides the response of the fingerprint Authentication.
+
+### iOS Biometrics
+
+For iOS devices, biometrics can be configured with different outputs. We support the following instance methods: 
+* [`evaluatePolicy`](https://developer.apple.com/documentation/localauthentication/lacontext/1514176-evaluatepolicy) evaluates the specified policy. The method [`canEvaluatePolicy`](https://developer.apple.com/documentation/localauthentication/lacontext/1514149-canevaluatepolicy) checks whether the authentication can proceed for a given policy.
+* [`biometryType`](https://developer.apple.com/documentation/localauthentication/lacontext/2867583-biometrytype) determines the type of biometric authentication method supported by the device and the method [`evaluatedPolicyDomainState`](https://developer.apple.com/documentation/localauthentication/lacontext/1514150-evaluatedpolicydomainstate) assesses the current state of the policy domain.
+* [`interactionNotAllowed`](https://developer.apple.com/documentation/localauthentication/lacontext/2873508-interactionnotallowed) indicates if the authentication can be interactive.
+* [`SecItemAdd`](https://developer.apple.com/documentation/security/1401659-secitemadd) adds one or more items to a keychain. [`SecItemCopyMatching`](https://developer.apple.com/documentation/security/1398306-secitemcopymatching)returns more keychain items that match the search query. You can delete items that match the search query by using [`SecItemDelete`](https://developer.apple.com/documentation/security/1395547-secitemdelete) method. We support the following SecItem Classes: [`kSecClassGenericPassword`](https://developer.apple.com/documentation/security/ksecclassgenericpassword) and [`kSecClassInternetPassword`](https://developer.apple.com/documentation/security/ksecclassinternetpassword).
+
+From API 23 to 28 we support the following:
```

Comment:
The linked docs indicate both of these methods were deprecated in API 28.  I usually read "to" as being inclusive. : ) 

```suggestion
From API 23 to 28 we support the following:
```

Reply:
@aliciakrebs  The development team confirmed that this method can be published, even if it is marked as deprecated on the android page, Sauce still supports it.